revscoring/languages/tests/test_german.py

```diff
@@ -0,0 +1,126 @@
+    "verarscht", "verfickte", "vollidiot", "wichser", "wixer", "wixxe", "wixxen", "wixxer"
+]
+
+INFORMAL = [
+    "anal", "auserdem", "autorenportal", "bins", "bla", "blabla", "blablabla", "blub", "blubb",
+    "blöd", "blöder", "bodewell", "bumsen", "cool", "coole", "cooler", "coolste", "coool", "deine",
+    "digga", "dildo", "dildos", "doof", "dumme", "dummen", "dummer", "dummes", "dummm", "döner",
+    "euch", "fetter", "fick", "ficke", "ficken", "fickt", "fickte", "fickten", "fresse", "fuck",
+    "furtz", "furz", "furzen", "fürn", "fürze", "gay", "gefickt", "gehts", "geil", "geile", "geilen",
+    "geiler", "geilste", "geilsten", "gez", "grüße", "hab", "haha", "hahah", "hahaha", "hahahah",
+    "hahahaha", "hahahahaha", "hahahahahaha", "hahahahahahaha", "halllo", "hallo", "halts", "hehe",
+    "hey", "hihi", "hihihi", "homos", "huhu", "hässlich", "jaja", "jannik", "juhu", "kack", "kacken",
+    "kackt", "kaka", "kake", "kaken", "klo", "kneipenschlägerein", "kotze", "kotzen", "kursiver",
+    "könnt", "labert", "lalala", "lalalala", "langhaardackel", "langweilig", "leck", "lecker", "leckt",
+    "lol", "lonni", "looser", "lutschen", "lutscht", "mama", "mfg", "minecraft", "moin", "mudda",
+    "mudder", "muhaha", "muhahaha", "mumu", "muschi", "muschie", "möse", "naja", "nich", "nix",
+    "noob", "nutten", "oma", "omg", "opfa", "penis", "penise", "penisen", "penises", "penner",
+    "pimmel", "pipi", "pisse", "popel", "popo", "porno", "pornos", "puff", "puffs", "pups", "pupsen",
+    "rofl", "schei", "scheis", "schlampen", "schniedel", "schwachsinn", "schwule", "seid", "sex", "sexy",
+    "shit", "soo", "sooo", "soooo", "sooooo", "spasti", "spezial", "stingt", "stink", "stinke", "stinken",
+    "stinker", "stinkst", "stinkt", "sucks", "swag", "titte", "titten", "tobi", "toll", "unformatierten",
+    "vaginas", "wisst", "xdd", "xddd", "xdddd", "xnxx", "yeah", "yolo", "youporn", "ärsche"
+]
+
+OTHER = [
+    """
+    The Smyth Report is the common name of an administrative history written
+    by physicist Henry DeWolf Smyth about the Manhattan Project, the Allied
+    effort to develop atomic bombs during World War II. It was released to the
+    public on August 12, 1945, just days after the atomic bombings of Hiroshima
+    and Nagasaki. Smyth was commissioned to write the report by Major General
+    Leslie Groves, the director of the Manhattan Project. The Smyth Report was
+    the first official account of the development of the atomic bombs and the
+    basic physical processes behind them. Since anything in the declassified
+    Smyth Report could be discussed openly, it focused heavily on basic nuclear
+    physics and other information which was either already widely known in the
+    scientific community or easily deducible by a competent scientist. It
+    omitted details about chemistry, metallurgy, and ordnance, ultimately
+    giving a false impression that the Manhattan Project was all about physics.
+    """,
```

Comment:
This block of text should be in german. I recommend copy-pasting from an article on the german wiki.